### PR TITLE
WooCommerce: Address outstanding design issues

### PR DESCRIPTION
**Pull request by jameskoster**
A list of smaller design issues that need fixing before release:

- [x] Additional details preview box doesn't quite align with inputs
- [x] Product list column header colors
- [x] Breadcrumb size / styling / give links hover cursor
- [x] Padding below Product Description in add-product
- [x] Inventory spacing on add product screen
- [x] Update all copy per Aviva's [copy review](https://docs.google.com/document/d/1iOqxh0FTFjbN_5tZj7KDOOg8-8MYBjEW0jLYq7p1Otw/edit) 
- [x] Add product logos to Stripe and Paypal on settings/payments [#](https://www.paypalobjects.com/webstatic/en_US/i/buttons/PP_logo_h_200x51.png) [#](https://stripe.com/about/resources)
- [x] Additional details description text looks a bit weird as a label
- [x] Make Fulfilled and Refunded status labels gray in order-list
- [x] Hover states for sharing icon links on the non-egg dashboards
- [x] Orders list link styles
- [x] Orders / Products list hover bg colors don't match
- [x] Product name line-height in order screen https://cloudup.com/c2m2WIXFphZ
- [x] Firefox issues

### CODE REVIEW

**Comment by jameskoster** [comment]
This is mostly styling but if @justinshreve could take a quick look at [how I'm adding payment method classes](https://github.com/Automattic/wp-calypso/pull/15494/commits/54a7b2a1bf18a96862314767bf99b80fee31eb99) it'd be appreciated.

**Comment by kellychoffman** [comment]
I found another thing for the list. The add product button on mobile:

screen shot 2017-06-27 at 7 48 57 am

**Comment by kellychoffman** [comment]
👍  Latest updates look good.

**Comment by justinshreve** [comment]
> This is mostly styling but if @justinshreve (or any other dev) could take a quick look at how I'm adding payment method classes it'd be appreciated.

That looks fine to me.

File: client/extensions/woocommerce/app/dashboard/style.scss [hunk 1]
```diff
@@ -219,8 +219,10 @@
 				margin-bottom: 0;
 				order: 1;
 				transform: translateY( 31px );
-				max-width: none;
 				margin-top: 0;
+				width: 360px;
+				height:180px;
```
Comment (justinshreve) [inline client/extensions/woocommerce/app/dashboard/style.scss]: Space

File: client/extensions/woocommerce/components/share-widget/style.scss [hunk 2]
```diff
@@ -36,6 +40,15 @@
 
 		li.share-widget__service {
 			margin: 0 4px;
+
+			a {
+				transition: transform ease-in-out .15s;
+				display: block;
+
+				&:hover {
+					transform: translateY(-4px) scale(1.05);
```
Comment (justinshreve) [inline client/extensions/woocommerce/components/share-widget/style.scss]: spaces inside the paren

**Pull request merged**